OpenRA.Mods.Cnc/Traits/SupportPowers/IonCannonPower.cs

```diff
@@ -50,7 +55,6 @@ public override IOrderGenerator OrderGenerator(string order, SupportPowerManager
 				manager.Self.World.Selection.Clear();
 
 			Sound.PlayToPlayer(manager.Self.Owner, Info.SelectTargetSound);
-			var info = Info as IonCannonPowerInfo;
 			return new SelectGenericPowerTarget(order, manager, info.Cursor, MouseButton.Left);
 		}
```

Comment:
You can indeed now remove this whole method. The selection is cleared again in the constructor of `SelectGenericPowerTarget`.